api/v1alpha1/openapi.yaml

```diff
@@ -4865,6 +4865,7 @@ components:
       - 'Synced'                # ResourceSync
       - 'OverlappingSelectors'  # Fleet
       - 'Valid'                 # Fleet
+      - 'Rollout'               # Fleet
```

Comment:
@fzdarsky Are you OK with this name?

Reply:
A condition type name should described the current observed state of the resource (here: the Fleet) and would be an adjective or past-tense verb. In the case of long running state transitions (like a Rollout), the transition itself is the thing whose state is being observed, though. A good test would be whether the question "is <Condition>...?" or "has <Condition>...?" and the answer "true/false/unknown" makes sense. For "Rollout" I don't think it does.

How about "RolloutInProgress" or - if we want to focus on the outcome - "RolloutCompleted" or "RolloutSucceeded"? There can also be multiple conditions if that makes more sense.

Reply:
Actually, this should have been a status, but since we want to be ready for other strategies, then the condition is a compromise.
So "RolloutInProgress" seems the best fit.